gradle/testing.gradle

```diff
@@ -282,7 +282,7 @@ tasks.withType(Test) { theTask ->
 }
 
 tasks.withType(Test).configureEach { task ->
-    if (task.name != 'performanceTest') {
+    if (task.name != 'performanceTest' && task.name != 'autoTest') {
```

Comment:
Is there a reason we can't validate async-to-sync calls in auto test? Does something break?

Reply:
No Idea, that wasn't what actually broke. I can't recall what failed.
That being said, I think it would be better to just add the AutoTestEngine to nightly, and not have a separate gradle testing task.